adminSiteClient/GdocsDateline.tsx

```diff
@@ -45,7 +46,7 @@ export const GdocsDateline = ({
                     <DatePicker
                         onChange={onChangePublishedAt}
                         value={
-                            publishedAt ? dayjs(publishedAt).utc() : undefined
+                            publishedAt ? dayjs(publishedAt).local() : undefined
```

Comment:
One evening when testing this, because I'm UTC-5, I set an article to publish "today" only to find the settings form datepicker showing that the published date was tomorrow (in UTC)

There's still some ambiguity around how dates are handled, but I think this was only confusing things.